.pre-commit-config.yaml

```diff
@@ -5,10 +5,10 @@ repos:
     hooks:
       - id: end-of-file-fixer
       - id: trailing-whitespace
-  - repo: https://github.com/norwoodj/helm-docs
-    rev: v1.11.0
-    hooks:
-      - id: helm-docs
+  # - repo: https://github.com/norwoodj/helm-docs
```

Comment:
need to uncomment this? this still works, right?

Reply:
It's not working for me locally, keeps trying to remove the `Autogenerated by ...` line. Does it work on your local?

Reply:
yeah it works fine for me. i have the following version in local. check if you have the same. 
```
➜ helm-docs --version
helm-docs version 1.11.0
```